client/app/_providers/reactQuery.tsx

```diff
@@ -0,0 +1,24 @@
+'use client'
+
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
+// import { ReactQueryDevtools } from '@tanstack/react-query-devtools'
+
+const ReactQueryProvider = ({ children }: { children: React.ReactNode }) => {
+  const queryClient = new QueryClient({
+    defaultOptions: {
+      queries: {
+        staleTime: Infinity,
+      },
+    },
+  })
+
+  return (
+    <QueryClientProvider client={queryClient}>
+      {/* NOTE If you need dev tools, turn on this  */}
+      {/* <ReactQueryDevtools initialIsOpen={false} /> */}
```

Comment:
I installed ReactQuery Devtools just in case. So if you want to use, just turn on this code!